cartridge/roles/metrics.lua

```diff
@@ -80,10 +72,16 @@ end
 local function validate_routes(export)
     local paths = {}
     for _, v in ipairs(export) do
+        if type(v.path) ~= 'string' then
+            error('path must be a string', 0)
```

Comment:
I suggest adding some info about the source of an error. E.g. `export[%d]` or something similar. At least it should specify that the problem is in the exports config, and not in `cartridge-extensions` (roughly speaking).